commands/sandbox/delete.ts

```diff
@@ -71,20 +80,30 @@ exports.handler = async options => {
 
   let parentAccountId;
   const accountsList = getConfigAccounts();
+  if (!accountsList) {
```

Comment:
If you do `const accountsList = getConfigAccounts() || []` above, the `for (const portal of accountsList)` will skip, causing `parentAccountId` to remain undefined. This would trigger the if check beneath it. Then you'd only need to log the "no parent Id" message in one place.